django/db/migrations/loader.py

```diff
@@ -6,11 +6,15 @@
 
 from django.apps import apps
 from django.conf import settings
+from django.db import connection as connection_
```

Comment:
We are you using this instead of `self.connection`? Will this work in the multi-database case?